jj/logs/formatters/_simple_formatter.py

```diff
@@ -1,21 +1,37 @@
+import os
 from logging import LogRecord
-from os import linesep
 
 from ...requests import Request
 from ...responses import Response
 from ._formatter import Formatter
 
 __all__ = ("SimpleFormatter",)
 
+INFO = 'info'
+DEBUG = 'debug'
+TRACE = 'trace'
+
+log_level = os.getenv('MOCK_LOG_LEVEL', INFO).lower()
+body_output_limit = int(os.getenv('MOCK_BODY_OUTPUT_LIMIT', 10 ** 6))
+
 
 class SimpleFormatter(Formatter):
-    def format_request(self, request: Request, record: LogRecord) -> str:
-        return "-> {} {}".format(request.method, request.url.path)
-
-    def format_response(self, response: Response, request: Request, record: LogRecord) -> str:
-        res = linesep.join([
-            "-> {} {}".format(request.method, request.url.path),
-            "<- {} {}".format(response.status, response.reason),
-            "",
-        ])
-        return res
+    def format_request(self, request: Request, record: LogRecord,
+                       log_level: str = log_level) -> str:
+        formatter = f"-> {request.method} {request.url.path_qs}"
+
+        if log_level == DEBUG or log_level == TRACE:
+            formatter += f"\n-> {dict(request.headers)}"
+
+        return formatter
+
+    def format_response(self, response: Response, request: Request, record: LogRecord,
+                        log_level: str = log_level) -> str:
+        formatter = f"<- {response.status} {response.reason}\n"
+
+        if log_level == DEBUG or log_level == TRACE:
```

Comment:
Typically, log levels are employed to dictate which messages get output, focusing more on the severity of the messages (DEBUG, INFO, WARNING, etc) rather than their verbosity or formatting.

For more granular control over message details, format specifications are better suited to this task, similar to what's implemented in [aiohttp](https://docs.aiohttp.org/en/v3.8.5/logging.html#format-specification).